src/index.d.ts

```diff
@@ -982,6 +982,121 @@ declare namespace admin.remoteConfig {
    */
   type RemoteConfigParameterValue = ExplicitParameterValue | InAppDefaultValue;
 
+  /**
+   * Interface representing a Remote Config template version.
+   * Contains all metadata about a particular version of the Remote Config template.
+   * All fields are set at the time the specified Remote Config template was written.
+   */
+  export interface Version {
+    /**
+     * The version number of the version's corresponding Remote Config template. Output only.
+     */
+    versionNumber?: string | number;
+    
+    /**
+     * The Remote Config template creation timestamp in RFC3339 UTC "Zulu" format,
+     * accurate to nanoseconds. Output only.
+     */
+    updateTime?: string;
+
+    /**
+     * The source of origin of the template update action. Output only.
+     */
+    updateOrigin?: ('REMOTE_CONFIG_UPDATE_ORIGIN_UNSPECIFIED' | 'CONSOLE' |
+      'REST_API' | 'ADMIN_SDK_NODE');
+
+    /**
+     * The type of the template update action. Output only.
+     */
+    updateType?: ('REMOTE_CONFIG_UPDATE_TYPE_UNSPECIFIED' |
+      'INCREMENTAL_UPDATE' | 'FORCED_UPDATE' | 'ROLLBACK');
+
+    /**
+     * Aggregation of all metadata fields about the account that performed the update. Output only.
+     */
+    updateUser?: RemoteConfigUser;
+
+    /**
+     * The user-provided description of the corresponding Remote Config template. Optional.
+     */
+    description?: string;
+
+    /**
+     * Only present if this version is the result of a rollback, and will be the version number of
+     * the Remote Config template that was rolled-back to. Output only.
+     */
+    rollbackSource?: string;
+    
+    /**
+     * Indicates weather this Remote Config template was published before version history was
+     * supported. Output only.
+     */
+    isLegacy?: boolean;
+  }
+
+  /** Interface representing a Remote Config list version result. */
+  export interface ListVersionsResult {
+    /**
+     * A list of version metadata objects, sorted in reverse chronological order.
+     */
+    versions: Version[];
+
+    /**
+     * Token to retrieve the next page of results, or empty if there are no more results
+     * in the list.
+     */
+    nextPageToken?: string;
+  }
+
+  /** Interface representing a Remote Config list version options. */
+  export interface ListVersionsOptions {
+    /**
+     * The maximum number of items to return per page.
+     */
+    pageSize?: number;
+
+    /**
+     * The `nextPageToken` value returned from a previous list versions request, if any.
+     */
+    pageToken?: string;
+    
+    /** 
+     * Specify the newest version number to include in the results.
+     * If specified, must be greater than zero. Defaults to the newest version.
+     */
+    endVersionNumber?: string | number;
+
+    /**
+     * Specify the earliest update time to include in the results. Any entries updated before this
+     * time are omitted.
+     */
+    startTime?: Date;
```

Comment:
Should we accept `Date | string` here? Our existing APIs (auth.UserMetadataRequest) accepts UTC formatted strings.

Reply:
Good point! Will update!